### PR TITLE
feat: support model name when creating model

### LINKED ISSUE #20: Container command is inconsistent
`model-backend` has a docker command "./openapi" at the moment. 

It should be `model-backend` to cohere other backends.

### LINKED ISSUE #21: Typo in README.md
https://github.com/instill-ai/model-backend/blob/9eb75d70b00198b6a0362e294104d07a082f7ba4/README.md?plain=1#L16

### LINKED ISSUE #22: Wrong model type
The model type is mistakenly set as `tensorrt`.
https://github.com/instill-ai/model-backend/blob/9eb75d70b00198b6a0362e294104d07a082f7ba4/examples-go/grpc_client.go#L65

### PR DESCRIPTION
**Pull request by Phelan164**
Because

- user wants to specify model name when they create a model and auto increase model version

This commit

- support model name and model version in CreateModel method (mapping model folder into Triton server based on namespace, model name, folder model, and model version)
- close #20
- close #21 
- close #22

### CODE REVIEW

**Comment by Phelan164** [comment]
close #22

**Comment by Phelan164** [comment]
close #20

File: README.md [hunk 1]
```diff
@@ -25,9 +24,9 @@ $ ./examples-go/quick-download.sh
 
 ```bash
 $ docker-compose up -d
-$ go run ./examples-go/grpc_client.go upload --file sample-models/yolov4-onnx-cpu.zip --cvtask 2  # upload a YOLOv4 model for object detection; note --cvtask 0: undefined 1: image classification task and 2: object detection task 
-$ go run ./examples-go/grpc_client.go load -n yolov4  # deploy the ensemble model
-$ go run ./examples-go/grpc_client.go predict -n yolov4 -f sample-models/dog.jpg # make inference
+$ go run ./examples-go/grpc_client.go upload --file sample-models/yolov4-onnx-cpu.zip --name yolov4 --version 1 --cvtask 2  # upload a YOLOv4 model for object detection; note --cvtask 0: undefined 1: image classification task and 2: object detection task
```
Comment (pinglin) [inline README.md:27]: Do we need to specify `cvtask`?

**Review by xiaofei-du — APPROVED** [review]
Just one question, I think we will make the docker image version of `model-backend` and `pipeline-backend` in the docker-compose.yml file fixed after we push official images to Docker Hub, right?

Other than that, LGTM 👍

**Review by xiaofei-du — DISMISSED** [review]

**Comment by Phelan164** [comment]
@xiaofei-du 
>Does user need to specify the version when creating a model?

Currently, user no need to specify version when creating a model. The model version will be auto increased.

**Review by xiaofei-du — APPROVED** [review]

**Pull request merged**